dev/com.ibm.ws.jdbc/resources/com/ibm/ws/rsadapter/resources/IBMDataStoreAdapterNLS.nlsprops

```diff
@@ -299,6 +299,17 @@ META_DATA_EXCEPTION.useraction=Verify that the metadata is enabled on the backen
 # 7019 deleted
 # 7020 reserved for other NLS file
 # 7021-7042 deleted
+
+# Do not translate: java.util.Logging
+ORACLE_TRACE_DUP_WARNING=DSRA7043I: Multiple DataSources attempted to configure custom Oracle logging properties, which is unsupported. The duplicate configuration was ignored.
+ORACLE_TRACE_DUP_WARNING.explanation=The Oracle JDBC driver uses the global java.util.Logging library to generate logs and does not allow for separate logging configurations per DataSource.
+ORACLE_TRACE_DUP_WARNING.useraction=Configure custom Oracle logging properties on exactly one DataSource.
```

Comment:
This is a design issue with this implementation. 
Really, this type of configuration should be happening at the library level, not the datasource level.
But, adding driver-specific logging properties to the library config element doesn't seem like a good design either. 

If the goal is to introduce this as a migration gap (since you can configure this on Traditional WebSphere) then I'd say I don't like it, but it's reasonable.  Otherwise, I'd like to see this put into its own configuration element.  

Maybe introduce something like,
```xml
    <jdbcLibrary id="DBLib">
    	<fileset dir="${shared.resource.dir}/oracle"/>
        <logging.properties 	    	
                fileSizeLimit="0"
    		fileCount="0"
    		fileName="oracle.log"
    		traceLevel="INFO"
    		formatter="java.util.SimpleFormatter"
    		packageName="oracle.jdbc" 
    </jdbcLibrary>
```

Reply:
The resolution to this design issue was to switch to looking for System properties.